Formula/clickhouse.rb

```diff
@@ -0,0 +1,74 @@
+class Clickhouse < Formula
+  desc "Free analytics DBMS for big data with SQL interface"
+  homepage "https://clickhouse.tech"
+  url "https://github.com/ClickHouse/ClickHouse/releases/download/v21.6.3.14-stable/ClickHouse_sources_with_submodules.tar.gz"
+  version "21.6"
+  sha256 "696bbcdef4796bfdf1becdb78abb5e97c0220acc9863f5a821193dd297ce428b"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/ClickHouse.git"
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on xcode: :build
+
+  def install
+    mkdir_p bin
+    mkdir_p "#{HOMEBREW_PREFIX}/etc/clickhouse-server"
+    mkdir_p "#{var}/log/clickhouse-server"
+    mkdir_p "#{var}/lib/clickhouse-server"
+    mkdir_p "#{var}/run/clickhouse-server"
```

Comment:
What happens when you don't do this?

Reply:
Install procedure will not find a directory.
At least I had to fix creation of ones like `Cellar/clickhouse/21.6/bin` that has version number in them.
By the way, it was really exhausting to debug build failures in `brew install --build-from-source -v clickhouse` as it required to rebuild everything without cache. And it took more than 3 hours on each try. Is there a way to retry one step in noninteractive way?